dockstore-client/src/main/java/io/dockstore/client/cli/YamlClient.java

```diff
@@ -0,0 +1,113 @@
+/*
+ *    Copyright 2022 OICR and UCSC
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package io.dockstore.client.cli;
+
+import java.util.List;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.ParameterException;
+import com.beust.jcommander.Parameters;
+import io.dockstore.client.cli.YamlVerify.ValidateYamlException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static io.dockstore.client.cli.ArgumentUtility.out;
+import static io.dockstore.client.cli.JCommanderUtility.printJCommanderHelp;
+import static io.dockstore.client.cli.JCommanderUtility.printJCommanderHelpYaml;
+import static io.dockstore.client.cli.YamlVerify.DOCKSTOREYML;
+import static io.dockstore.client.cli.YamlVerify.YAML;
+
+/**
+ *
+ */
+public final class YamlClient {
+
+    public static final String NO_PATH_FLAG = "ERROR: Missing --path <path>";
+    public static final String VALIDATE_HELP_MESSAGE = "Verifies " + DOCKSTOREYML + " is correct and ensures all required files are present";
+    public static final String PATH_ON_COMPUTER = "Path on computer (ex. /home/usr/Dockstore/test, ~/Dockstore/test, or ../test)";
```

Comment:
1. "Path on computer" -- "on computer" seems redundant
2. What path is it? If it's a .dockstore.yml, then use either use that as an example (/home/usr/Dockstore/.dockstore.yml), or if it's the directory of the .dockstore.yml, say that. Or of it's something else entirely, describe that. But I feel "path" is too general.

Reply:
> This doesn't build for me locally, and it looks like the CI builds failed too, so I'm going to stop reviewing -- I'd like to be able to navigate in the IDE.

It appears that the  merging of this [PR](https://github.com/dockstore/dockstore-cli/pull/165) caused some issues. I Have created this [PR](https://github.com/dockstore/dockstore/pull/5004/files) to correct that issue.

Reply:
I have changed my wording of "Path on computer" to `"Path to .dockstore.yml (ex. /home/usr/Dockstore/test, ~/Dockstore/test, or ../test)"`